2025-05-10 18:29:46,668 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 18:29:46,669 [ERROR] Socket failed to connect: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 36320, 0, 0)>; error=111 (Connection refused)
2025-05-10 18:29:46,669 [ERROR] TCP Connection attempt failed: ConnectionRefusedError(111, 'Connection refused'); dest=(<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 5672, 0, 0))
2025-05-10 18:29:46,669 [ERROR] AMQPConnector - reporting failure: AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused')
2025-05-10 18:29:46,669 [INFO] Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-05-10 18:29:46,670 [ERROR] Socket failed to connect: <socket.socket fd=10, family=2, type=1, proto=6, laddr=('127.0.0.1', 54224)>; error=111 (Connection refused)
2025-05-10 18:29:46,670 [ERROR] TCP Connection attempt failed: ConnectionRefusedError(111, 'Connection refused'); dest=(<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('127.0.0.1', 5672))
2025-05-10 18:29:46,670 [ERROR] AMQPConnector - reporting failure: AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused')
2025-05-10 18:29:46,670 [ERROR] AMQP connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused'); first exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused').
2025-05-10 18:29:46,670 [ERROR] AMQPConnectionWorkflow - reporting failure: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused'); first exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused')
2025-05-10 18:29:46,671 [ERROR] Connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused'); first exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused')
2025-05-10 18:29:46,671 [ERROR] Error in _create_connection().
Traceback (most recent call last):
  File "/home/milax/SD/sd_p1/dynamic-scaling/venv/lib/python3.11/site-packages/pika/adapters/blocking_connection.py", line 451, in _create_connection
    raise self._reap_last_connection_workflow_error(error)
pika.exceptions.AMQPConnectionError
2025-05-10 18:29:46,672 [INFO] InsultFilterService_0 registered at PYRO:obj_a951671621154d45ba31f7d4cee4094c@localhost:50152
2025-05-10 19:05:48,628 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 19:05:48,629 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50990, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 19:05:48,630 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f74ca5b8110>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f74ca5b8110> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 19:05:48,631 [ERROR] Socket EOF; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50990, 0, 0)>
2025-05-10 19:05:48,631 [ERROR] Transport indicated EOF.
2025-05-10 19:05:48,631 [INFO] protocol.eof_received() elected to close: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50990, 0, 0)>
2025-05-10 19:05:48,631 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50990, 0, 0)>
2025-05-10 19:05:48,631 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50990, 0, 0)>
2025-05-10 19:05:48,631 [ERROR] connection_lost: StreamLostError: ('Transport indicated EOF',)
2025-05-10 19:05:48,632 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=False, error-arg=StreamLostError: ('Transport indicated EOF',); pending-error=None
2025-05-10 19:05:48,632 [ERROR] Probably incompatible Protocol Versions
2025-05-10 19:05:48,632 [INFO] Connection setup terminated due to IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:05:48,632 [ERROR] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:05:48,632 [INFO] Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-05-10 19:05:48,632 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50990, 0, 0)>
2025-05-10 19:05:48,633 [INFO] InsultFilterService_0 registered at PYRO:obj_cec5ff9863274b9e9fbf96d1c45b1854@localhost:50152
2025-05-10 19:05:48,633 [INFO] Socket connected: <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 34518), raddr=('127.0.0.1', 5672)>
2025-05-10 19:05:48,633 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f74ca5b9350>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f74ca5b9350> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 19:05:48,634 [ERROR] Socket EOF; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 34518)>
2025-05-10 19:05:48,634 [ERROR] Transport indicated EOF.
2025-05-10 19:05:48,634 [INFO] protocol.eof_received() elected to close: <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 34518)>
2025-05-10 19:05:48,634 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 34518)>
2025-05-10 19:05:48,634 [INFO] Deactivating transport: state=1; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 34518)>
2025-05-10 19:05:48,635 [ERROR] connection_lost: StreamLostError: ('Transport indicated EOF',)
2025-05-10 19:05:48,635 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=False, error-arg=StreamLostError: ('Transport indicated EOF',); pending-error=None
2025-05-10 19:05:48,635 [ERROR] Probably incompatible Protocol Versions
2025-05-10 19:05:48,635 [INFO] Connection setup terminated due to IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:05:48,635 [ERROR] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:05:48,637 [ERROR] AMQP connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",); first exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",).
2025-05-10 19:05:48,637 [ERROR] AMQPConnectionWorkflow - reporting failure: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",); first exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:05:48,637 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 34518)>
2025-05-10 19:05:48,637 [ERROR] Connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",); first exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:05:48,638 [ERROR] Error in _create_connection().
Traceback (most recent call last):
  File "/home/milax/SD/sd_p1/dynamic-scaling/venv/lib/python3.11/site-packages/pika/adapters/blocking_connection.py", line 451, in _create_connection
    raise self._reap_last_connection_workflow_error(error)
pika.exceptions.IncompatibleProtocolError: StreamLostError: ('Transport indicated EOF',)
2025-05-10 19:06:41,444 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 19:06:41,445 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 38034, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 19:14:02,107 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 19:14:02,107 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 54716, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 19:14:02,108 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7430db8210>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7430db8210> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 19:14:02,109 [ERROR] Socket EOF; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 54716, 0, 0)>
2025-05-10 19:14:02,109 [ERROR] Transport indicated EOF.
2025-05-10 19:14:02,109 [INFO] protocol.eof_received() elected to close: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 54716, 0, 0)>
2025-05-10 19:14:02,109 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 54716, 0, 0)>
2025-05-10 19:14:02,109 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 54716, 0, 0)>
2025-05-10 19:14:02,110 [ERROR] connection_lost: StreamLostError: ('Transport indicated EOF',)
2025-05-10 19:14:02,110 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=False, error-arg=StreamLostError: ('Transport indicated EOF',); pending-error=None
2025-05-10 19:14:02,110 [ERROR] Probably incompatible Protocol Versions
2025-05-10 19:14:02,110 [INFO] Connection setup terminated due to IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:14:02,110 [ERROR] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:14:02,110 [INFO] InsultFilterService_0 registered at PYRO:obj_d700e06a3f024f0d826afcab3a9a65d4@localhost:50152
2025-05-10 19:14:02,110 [INFO] Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-05-10 19:14:02,111 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 54716, 0, 0)>
2025-05-10 19:14:02,111 [INFO] Socket connected: <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 38218), raddr=('127.0.0.1', 5672)>
2025-05-10 19:14:02,111 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7430db9490>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7430db9490> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 19:14:02,112 [ERROR] Socket EOF; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 38218)>
2025-05-10 19:14:02,112 [ERROR] Transport indicated EOF.
2025-05-10 19:14:02,112 [INFO] protocol.eof_received() elected to close: <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 38218)>
2025-05-10 19:14:02,112 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 38218)>
2025-05-10 19:14:02,112 [INFO] Deactivating transport: state=1; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 38218)>
2025-05-10 19:14:02,112 [ERROR] connection_lost: StreamLostError: ('Transport indicated EOF',)
2025-05-10 19:14:02,112 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=False, error-arg=StreamLostError: ('Transport indicated EOF',); pending-error=None
2025-05-10 19:14:02,112 [ERROR] Probably incompatible Protocol Versions
2025-05-10 19:14:02,112 [INFO] Connection setup terminated due to IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:14:02,113 [ERROR] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:14:02,113 [ERROR] AMQP connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",); first exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",).
2025-05-10 19:14:02,113 [ERROR] AMQPConnectionWorkflow - reporting failure: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",); first exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:14:02,113 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 38218)>
2025-05-10 19:14:02,113 [ERROR] Connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",); first exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:14:02,113 [ERROR] Error in _create_connection().
Traceback (most recent call last):
  File "/home/milax/SD/sd_p1/dynamic-scaling/venv/lib/python3.11/site-packages/pika/adapters/blocking_connection.py", line 451, in _create_connection
    raise self._reap_last_connection_workflow_error(error)
pika.exceptions.IncompatibleProtocolError: StreamLostError: ('Transport indicated EOF',)
2025-05-10 19:21:36,276 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 19:21:36,278 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 35634, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 19:21:36,279 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fd30eb004d0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fd30eb004d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 19:21:36,284 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fd30eb004d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 19:21:36,284 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fd30eb004d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 19:21:36,284 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fd30eb004d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 19:21:36,284 [INFO] Created channel=1
2025-05-10 19:21:36,294 [INFO] InsultFilterService_0 registered at PYRO:obj_80cadf1f22864cc792b27d30afb21d4a@localhost:50152
2025-05-10 19:22:46,232 [INFO] Filtered text added: you're such a clown
2025-05-10 19:22:46,234 [INFO] Filtered text added: that was a moron move
2025-05-10 19:22:46,236 [INFO] Filtered text added: twit behavior again?
2025-05-10 19:22:46,238 [INFO] Filtered text added: what a loser you are
2025-05-10 19:40:00,836 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 19:40:00,837 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 54996, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 19:40:00,838 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fbe72158310>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fbe72158310> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 19:40:00,838 [ERROR] Socket EOF; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 54996, 0, 0)>
2025-05-10 19:40:00,839 [ERROR] Transport indicated EOF.
2025-05-10 19:40:00,839 [INFO] protocol.eof_received() elected to close: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 54996, 0, 0)>
2025-05-10 19:40:00,839 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 54996, 0, 0)>
2025-05-10 19:40:00,839 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 54996, 0, 0)>
2025-05-10 19:40:00,839 [ERROR] connection_lost: StreamLostError: ('Transport indicated EOF',)
2025-05-10 19:40:00,839 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=False, error-arg=StreamLostError: ('Transport indicated EOF',); pending-error=None
2025-05-10 19:40:00,839 [ERROR] Probably incompatible Protocol Versions
2025-05-10 19:40:00,839 [INFO] Connection setup terminated due to IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:40:00,840 [ERROR] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:40:00,840 [INFO] Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-05-10 19:40:00,840 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 54996, 0, 0)>
2025-05-10 19:40:00,841 [INFO] Socket connected: <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 34318), raddr=('127.0.0.1', 5672)>
2025-05-10 19:40:00,841 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fbe72159590>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fbe72159590> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 19:40:00,842 [ERROR] Socket EOF; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 34318)>
2025-05-10 19:40:00,842 [ERROR] Transport indicated EOF.
2025-05-10 19:40:00,842 [INFO] protocol.eof_received() elected to close: <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 34318)>
2025-05-10 19:40:00,842 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 34318)>
2025-05-10 19:40:00,842 [INFO] Deactivating transport: state=1; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 34318)>
2025-05-10 19:40:00,842 [ERROR] connection_lost: StreamLostError: ('Transport indicated EOF',)
2025-05-10 19:40:00,842 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=False, error-arg=StreamLostError: ('Transport indicated EOF',); pending-error=None
2025-05-10 19:40:00,843 [ERROR] Probably incompatible Protocol Versions
2025-05-10 19:40:00,843 [INFO] Connection setup terminated due to IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:40:00,843 [ERROR] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:40:00,843 [ERROR] AMQP connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",); first exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",).
2025-05-10 19:40:00,843 [ERROR] AMQPConnectionWorkflow - reporting failure: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",); first exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:40:00,843 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 34318)>
2025-05-10 19:40:00,843 [ERROR] Connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",); first exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 19:40:00,843 [ERROR] Error in _create_connection().
Traceback (most recent call last):
  File "/home/milax/SD/sd_p1/dynamic-scaling/venv/lib/python3.11/site-packages/pika/adapters/blocking_connection.py", line 451, in _create_connection
    raise self._reap_last_connection_workflow_error(error)
pika.exceptions.IncompatibleProtocolError: StreamLostError: ('Transport indicated EOF',)
2025-05-10 19:40:00,850 [INFO] InsultFilterService_0 registered at PYRO:obj_c0818233483a400d90ffbf565d65c33b@localhost:50152
2025-05-10 19:41:25,524 [INFO] InsultFilterService_0 registered at PYRO:obj_d0a77abb91084fffb25d3a0d9e59f565@localhost:50152
2025-05-10 19:41:25,524 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 19:41:25,525 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 45146, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 19:41:25,526 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2522100350>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2522100350> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 19:41:25,533 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 45146, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 19:41:25,534 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 45146, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 19:41:25,534 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 45146, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 19:41:25,534 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=False, error-arg=None; pending-error=ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'
2025-05-10 19:41:25,534 [ERROR] Connection closed while authenticating indicating a probable authentication error
2025-05-10 19:41:25,534 [INFO] Connection setup terminated due to ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2025-05-10 19:41:25,534 [ERROR] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2025-05-10 19:41:25,535 [INFO] Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-05-10 19:41:25,535 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 45146, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 19:41:25,538 [INFO] Socket connected: <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 52846), raddr=('127.0.0.1', 5672)>
2025-05-10 19:41:25,539 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2522101c90>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2522101c90> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 19:41:25,541 [INFO] Aborting transport connection: state=1; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 52846), raddr=('127.0.0.1', 5672)>
2025-05-10 19:41:25,542 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 52846), raddr=('127.0.0.1', 5672)>
2025-05-10 19:41:25,542 [INFO] Deactivating transport: state=1; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 52846), raddr=('127.0.0.1', 5672)>
2025-05-10 19:41:25,542 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=False, error-arg=None; pending-error=ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'
2025-05-10 19:41:25,542 [ERROR] Connection closed while authenticating indicating a probable authentication error
2025-05-10 19:41:25,542 [INFO] Connection setup terminated due to ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2025-05-10 19:41:25,542 [ERROR] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2025-05-10 19:41:25,543 [ERROR] AMQP connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",); first exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",).
2025-05-10 19:41:25,543 [ERROR] AMQPConnectionWorkflow - reporting failure: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",); first exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2025-05-10 19:41:25,544 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 52846), raddr=('127.0.0.1', 5672)>
2025-05-10 19:41:25,545 [ERROR] Connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",); first exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2025-05-10 19:41:25,545 [ERROR] Error in _create_connection().
Traceback (most recent call last):
  File "/home/milax/SD/sd_p1/dynamic-scaling/venv/lib/python3.11/site-packages/pika/adapters/blocking_connection.py", line 451, in _create_connection
    raise self._reap_last_connection_workflow_error(error)
pika.exceptions.ProbableAuthenticationError: ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'
2025-05-10 21:04:18,068 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 21:04:18,069 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 39690, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:04:18,071 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f8530894150>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f8530894150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 21:04:18,077 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f8530894150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:04:18,077 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f8530894150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:04:18,078 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f8530894150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:04:18,078 [INFO] Created channel=1
2025-05-10 21:04:18,710 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 39690, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:04:18,710 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 39690, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:04:18,710 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 39690, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:04:18,711 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:04:18,711 [INFO] Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:04:18,712 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 39690, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:04:18,712 [ERROR] Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:06:27,653 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 21:06:27,654 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 57456, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:06:27,656 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2030eb00d0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2030eb00d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 21:06:27,663 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2030eb00d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:06:27,663 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2030eb00d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:06:27,664 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2030eb00d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:06:27,665 [INFO] Created channel=1
2025-05-10 21:06:28,274 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 57456, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:06:28,275 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 57456, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:06:28,275 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 57456, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:06:28,276 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:06:28,285 [INFO] Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:06:28,285 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 57456, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:06:28,286 [ERROR] Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:07:33,987 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 21:07:33,988 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 43250, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:07:33,989 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fd6841e83d0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fd6841e83d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 21:07:33,993 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 43250, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:07:33,993 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 43250, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:07:33,993 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 43250, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:07:33,993 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=False, error-arg=None; pending-error=ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'
2025-05-10 21:07:33,993 [ERROR] Connection closed while authenticating indicating a probable authentication error
2025-05-10 21:07:33,993 [INFO] InsultFilterService_0 registered at PYRO:obj_fb4c8de4d5c24f37a4ee828058f86609@localhost:50152
2025-05-10 21:07:33,993 [INFO] Connection setup terminated due to ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2025-05-10 21:07:33,994 [ERROR] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2025-05-10 21:07:33,994 [INFO] Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-05-10 21:07:33,994 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 43250, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:07:33,994 [INFO] Socket connected: <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 60818), raddr=('127.0.0.1', 5672)>
2025-05-10 21:07:33,995 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fd6841e9d50>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fd6841e9d50> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 21:07:34,000 [INFO] Aborting transport connection: state=1; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 60818), raddr=('127.0.0.1', 5672)>
2025-05-10 21:07:34,000 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 60818), raddr=('127.0.0.1', 5672)>
2025-05-10 21:07:34,001 [INFO] Deactivating transport: state=1; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 60818), raddr=('127.0.0.1', 5672)>
2025-05-10 21:07:34,001 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=False, error-arg=None; pending-error=ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'
2025-05-10 21:07:34,001 [ERROR] Connection closed while authenticating indicating a probable authentication error
2025-05-10 21:07:34,001 [INFO] Connection setup terminated due to ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2025-05-10 21:07:34,001 [ERROR] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2025-05-10 21:07:34,001 [ERROR] AMQP connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",); first exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",).
2025-05-10 21:07:34,002 [ERROR] AMQPConnectionWorkflow - reporting failure: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",); first exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2025-05-10 21:07:34,002 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 60818), raddr=('127.0.0.1', 5672)>
2025-05-10 21:07:34,002 [ERROR] Connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",); first exception - AMQPConnectorAMQPHandshakeError: ProbableAuthenticationError: Client was disconnected at a connection stage indicating a probable authentication error: ("ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'",)
2025-05-10 21:07:34,003 [ERROR] Error in _create_connection().
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/pika/adapters/blocking_connection.py", line 451, in _create_connection
    raise self._reap_last_connection_workflow_error(error)
pika.exceptions.ProbableAuthenticationError: ConnectionClosedByBroker: (403) 'ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.'
2025-05-10 21:11:38,284 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 21:11:38,285 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 55278, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:11:38,287 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fa3fb03c150>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fa3fb03c150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 21:11:38,293 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fa3fb03c150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:11:38,293 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fa3fb03c150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:11:38,294 [INFO] InsultFilterService_0 registered at PYRO:obj_7a1f74aaaa91461e8bc30fb58ede7b07@localhost:50152
2025-05-10 21:11:38,294 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fa3fb03c150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:11:38,296 [INFO] Created channel=1
2025-05-10 21:14:31,009 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 55278, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:14:31,009 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 55278, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:14:31,010 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 55278, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:14:31,010 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:14:31,011 [INFO] Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:14:31,012 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 55278, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:14:31,014 [ERROR] Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:15:50,314 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 21:15:50,316 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50664, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:15:50,318 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2d4f0504d0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2d4f0504d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 21:15:50,323 [INFO] InsultFilterService_0 registered at PYRO:obj_fcca5d8c7e754f8b8424415d490ea627@localhost:50152
2025-05-10 21:15:50,331 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2d4f0504d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:15:50,332 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2d4f0504d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:15:50,332 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2d4f0504d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:15:50,332 [INFO] Created channel=1
2025-05-10 21:15:51,005 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50664, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:15:51,006 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50664, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:15:51,006 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50664, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:15:51,007 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:15:51,008 [INFO] Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:15:51,008 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50664, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:15:51,009 [ERROR] Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:19:52,180 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 21:19:52,181 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 35318, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:19:52,183 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fa8218943d0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fa8218943d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 21:19:52,188 [INFO] InsultFilterService_0 registered at PYRO:obj_cd1f234e893c41df921f0cafc9f9e678@localhost:50152
2025-05-10 21:19:52,188 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fa8218943d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:19:52,188 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fa8218943d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:19:52,189 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fa8218943d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:19:52,189 [INFO] Created channel=1
2025-05-10 21:19:54,881 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 35318, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:19:54,881 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 35318, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:19:54,881 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 35318, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:19:54,882 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:19:54,883 [INFO] Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:19:54,883 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 35318, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:19:54,884 [ERROR] Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:21:18,011 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 21:21:18,012 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 35032, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:21:18,014 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f05184942d0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f05184942d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 21:21:18,018 [INFO] InsultFilterService_0 registered at PYRO:obj_6e97dcfcf8ef454fa411c194b632eaae@localhost:50152
2025-05-10 21:21:18,018 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f05184942d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:21:18,019 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f05184942d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:21:18,019 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f05184942d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:21:18,020 [INFO] Created channel=1
2025-05-10 21:21:20,429 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 35032, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:21:20,429 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 35032, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:21:20,429 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 35032, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:21:20,429 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:21:20,429 [INFO] Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:21:20,430 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 35032, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:21:20,430 [ERROR] Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:27:19,945 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 21:27:19,946 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 56550, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:27:19,947 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fda2e63bfd0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fda2e63bfd0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 21:27:19,948 [INFO] InsultFilterService_0 registered at PYRO:obj_2f7d4220f9e74306bab6824ca2f63080@localhost:50152
2025-05-10 21:27:19,951 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fda2e63bfd0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:27:19,951 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fda2e63bfd0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:27:19,952 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fda2e63bfd0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:27:19,954 [INFO] Created channel=1
2025-05-10 21:27:27,169 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 56550, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:27:27,170 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 56550, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:27:27,170 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 56550, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:27:27,170 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:27:27,170 [INFO] Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:27:27,170 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 56550, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:27:27,170 [ERROR] Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 21:32:43,355 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 21:32:43,356 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 55658, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:32:43,358 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f3e829fc510>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f3e829fc510> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 21:32:43,365 [INFO] InsultFilterService_0 registered at PYRO:obj_113c0f45de9141d4ae36fa590885e23c@localhost:50152
2025-05-10 21:32:43,366 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f3e829fc510> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:32:43,366 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f3e829fc510> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:32:43,366 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f3e829fc510> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:32:43,367 [INFO] Created channel=1
2025-05-10 21:44:05,010 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 21:44:05,011 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 40246, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 21:44:05,012 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f62d3724090>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f62d3724090> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 21:44:05,019 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f62d3724090> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:44:05,020 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f62d3724090> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:44:05,020 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f62d3724090> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 21:44:05,021 [INFO] Created channel=1
2025-05-10 21:44:05,037 [INFO] InsultFilterService_0 registered at PYRO:obj_22b93e3e5eb54d848fb81bb848d56acc@localhost:50152
2025-05-10 22:18:24,522 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 22:18:24,523 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50488, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:18:24,525 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fe0dc5b4110>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fe0dc5b4110> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 22:18:24,531 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fe0dc5b4110> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:18:24,531 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fe0dc5b4110> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:18:24,531 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fe0dc5b4110> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:18:24,534 [INFO] Created channel=1
2025-05-10 22:18:24,549 [INFO] InsultFilterService_0 registered at PYRO:obj_b8bca80b11ac40d7b164054f43396920@localhost:50152
2025-05-10 22:20:04,434 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50488, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:20:04,434 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50488, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:20:04,434 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50488, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:20:04,435 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 22:20:04,435 [INFO] Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 22:20:04,435 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 50488, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:20:04,435 [ERROR] Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 22:23:27,177 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 22:23:27,178 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 39818, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:23:27,180 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9722bfc950>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9722bfc950> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 22:23:27,186 [INFO] InsultFilterService_0 registered at PYRO:obj_bd9743126ef44d809a1bb39ff22d2d27@localhost:50152
2025-05-10 22:23:27,187 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9722bfc950> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:23:27,187 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9722bfc950> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:23:27,188 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9722bfc950> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:23:27,188 [INFO] Created channel=1
2025-05-10 22:29:20,244 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 22:29:20,246 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 59020, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:29:20,249 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2ed15580d0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2ed15580d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 22:29:20,251 [INFO] InsultFilterService_0 registered at PYRO:obj_68c93688d75e49188c6d8fbdb29e3a24@localhost:50152
2025-05-10 22:29:20,267 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2ed15580d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:29:20,268 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2ed15580d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:29:20,268 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f2ed15580d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:29:20,269 [INFO] Created channel=1
2025-05-10 22:32:15,125 [INFO] InsultFilterService_0 registered at PYRO:obj_8a957d25aa44476e849ce77fad84a4cc@localhost:50152
2025-05-10 22:32:15,125 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 22:32:15,128 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 58372, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:32:15,135 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7be49fff10>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7be49fff10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 22:32:15,159 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7be49fff10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:32:15,160 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7be49fff10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:32:15,160 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7be49fff10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:32:15,161 [INFO] Created channel=1
2025-05-10 22:33:54,916 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 58372, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:33:54,917 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 58372, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:33:54,917 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 58372, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:33:54,917 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 22:33:54,918 [INFO] Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 22:33:54,918 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 58372, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:33:54,918 [ERROR] Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 22:39:21,517 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 22:39:21,518 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 58762, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:39:21,520 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f4ae9d58110>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f4ae9d58110> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 22:39:21,524 [INFO] InsultFilterService_0 registered at PYRO:obj_064a3e1add55497e8046a1a3e221d7ca@localhost:50152
2025-05-10 22:39:21,526 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f4ae9d58110> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:39:21,527 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f4ae9d58110> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:39:21,527 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f4ae9d58110> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:39:21,528 [INFO] Created channel=1
2025-05-10 22:41:02,483 [INFO] InsultFilterService_0 registered at PYRO:obj_8b216e68e18148b598f3f798cc4ac964@localhost:50152
2025-05-10 22:41:02,510 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 22:41:02,513 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 38472, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:41:02,516 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f25704940d0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f25704940d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 22:41:02,535 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f25704940d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:41:02,536 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f25704940d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:41:02,537 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f25704940d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:41:02,538 [INFO] Created channel=1
2025-05-10 22:50:50,769 [INFO] InsultFilterService_0 registered at PYRO:obj_e791dc9bd26346bfa0cc530e173ecaca@localhost:50152
2025-05-10 22:50:50,775 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 22:50:50,776 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 41818, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:50:50,780 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f970d9b4150>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f970d9b4150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 22:50:50,809 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f970d9b4150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:50:50,810 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f970d9b4150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:50:50,810 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f970d9b4150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:50:50,812 [INFO] Created channel=1
2025-05-10 22:54:49,164 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 22:54:49,167 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 56394, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 22:54:49,170 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7655b58190>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7655b58190> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 22:54:49,185 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7655b58190> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:54:49,187 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7655b58190> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:54:49,188 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7655b58190> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 22:54:49,191 [INFO] Created channel=1
2025-05-10 22:54:49,192 [INFO] InsultFilterService_0 registered at PYRO:obj_2541820c0a15494aae62ef650eadf9a8@localhost:50152
2025-05-10 23:01:32,238 [INFO] InsultFilterService_0 registered at PYRO:obj_1f4732abb52942d4871f59c4bc1d245c@localhost:50152
2025-05-10 23:01:32,248 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 23:01:32,249 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 45980, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 23:01:32,251 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9151a63d10>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9151a63d10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 23:01:32,259 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9151a63d10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:01:32,260 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9151a63d10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:01:32,260 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9151a63d10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:01:32,261 [INFO] Created channel=1
2025-05-10 23:03:23,686 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 23:03:23,688 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 38186, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 23:03:23,690 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fce6ac94190>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fce6ac94190> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 23:03:23,696 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fce6ac94190> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:03:23,697 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fce6ac94190> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:03:23,697 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fce6ac94190> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:03:23,698 [INFO] Created channel=1
2025-05-10 23:03:23,706 [INFO] InsultFilterService_0 registered at PYRO:obj_0d171479949342159f1217b912394145@localhost:50152
2025-05-10 23:10:27,651 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 23:10:27,652 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 37658, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 23:10:27,655 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f90a875c490>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f90a875c490> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 23:10:27,662 [INFO] InsultFilterService_0 registered at PYRO:obj_4c4766764a38467d92d101516b55c9db@localhost:50152
2025-05-10 23:10:27,664 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f90a875c490> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:10:27,665 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f90a875c490> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:10:27,665 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f90a875c490> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:10:27,666 [INFO] Created channel=1
2025-05-10 23:11:44,323 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 23:11:44,324 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 56320, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 23:11:44,326 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f4b33c941d0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f4b33c941d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 23:11:44,344 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f4b33c941d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:11:44,344 [INFO] InsultFilterService_0 registered at PYRO:obj_a2062f5b4b6a4425a2a79a2aec4c6a4b@localhost:50152
2025-05-10 23:11:44,346 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f4b33c941d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:11:44,346 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f4b33c941d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:11:44,347 [INFO] Created channel=1
2025-05-10 23:13:17,829 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 23:13:17,829 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 39334, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 23:13:17,831 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f5a8573c150>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f5a8573c150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 23:13:17,836 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f5a8573c150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:13:17,837 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f5a8573c150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:13:17,837 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f5a8573c150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:13:17,838 [INFO] Created channel=1
2025-05-10 23:13:17,838 [INFO] InsultFilterService_0 registered at PYRO:obj_ea81dafbf1164f22a7033b826e538994@localhost:50152
2025-05-10 23:16:49,785 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 23:16:49,786 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 48456, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 23:16:49,788 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fc3ca41ff10>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fc3ca41ff10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 23:16:49,790 [ERROR] Socket EOF; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 48456, 0, 0)>
2025-05-10 23:16:49,790 [ERROR] Transport indicated EOF.
2025-05-10 23:16:49,790 [INFO] protocol.eof_received() elected to close: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 48456, 0, 0)>
2025-05-10 23:16:49,792 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 48456, 0, 0)>
2025-05-10 23:16:49,795 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 48456, 0, 0)>
2025-05-10 23:16:49,799 [ERROR] connection_lost: StreamLostError: ('Transport indicated EOF',)
2025-05-10 23:16:49,799 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=False, error-arg=StreamLostError: ('Transport indicated EOF',); pending-error=None
2025-05-10 23:16:49,800 [ERROR] Probably incompatible Protocol Versions
2025-05-10 23:16:49,800 [INFO] Connection setup terminated due to IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 23:16:49,801 [ERROR] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 23:16:49,803 [INFO] Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-05-10 23:16:49,806 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 48456, 0, 0)>
2025-05-10 23:16:49,808 [INFO] Socket connected: <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 44982), raddr=('127.0.0.1', 5672)>
2025-05-10 23:16:49,826 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fc3ca43d1d0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fc3ca43d1d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 23:16:49,829 [ERROR] Socket EOF; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 44982)>
2025-05-10 23:16:49,830 [ERROR] Transport indicated EOF.
2025-05-10 23:16:49,830 [INFO] protocol.eof_received() elected to close: <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 44982)>
2025-05-10 23:16:49,831 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 44982)>
2025-05-10 23:16:49,832 [INFO] Deactivating transport: state=1; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 44982)>
2025-05-10 23:16:49,832 [ERROR] connection_lost: StreamLostError: ('Transport indicated EOF',)
2025-05-10 23:16:49,833 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=False, error-arg=StreamLostError: ('Transport indicated EOF',); pending-error=None
2025-05-10 23:16:49,834 [ERROR] Probably incompatible Protocol Versions
2025-05-10 23:16:49,834 [INFO] Connection setup terminated due to IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 23:16:49,835 [ERROR] AMQPConnector - reporting failure: AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 23:16:49,848 [ERROR] AMQP connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",); first exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",).
2025-05-10 23:16:49,850 [ERROR] AMQPConnectionWorkflow - reporting failure: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",); first exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 23:16:49,850 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=11, family=2, type=1, proto=6, laddr=('127.0.0.1', 44982)>
2025-05-10 23:16:49,850 [ERROR] Connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",); first exception - AMQPConnectorAMQPHandshakeError: IncompatibleProtocolError: The protocol returned by the server is not supported: ("StreamLostError: ('Transport indicated EOF',)",)
2025-05-10 23:16:49,851 [ERROR] Error in _create_connection().
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/pika/adapters/blocking_connection.py", line 451, in _create_connection
    raise self._reap_last_connection_workflow_error(error)
pika.exceptions.IncompatibleProtocolError: StreamLostError: ('Transport indicated EOF',)
2025-05-10 23:17:05,285 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 23:17:05,286 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 48812, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 23:17:05,287 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f820541ff10>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f820541ff10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 23:17:05,291 [INFO] InsultFilterService_0 registered at PYRO:obj_929fd1af2a0c4d3687a560faa36a79ae@localhost:50152
2025-05-10 23:17:05,294 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f820541ff10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:17:05,294 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f820541ff10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:17:05,295 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f820541ff10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:17:05,295 [INFO] Created channel=1
2025-05-10 23:20:20,321 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 48812, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 23:20:20,322 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 48812, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 23:20:20,322 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 48812, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 23:20:20,322 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 23:20:20,322 [INFO] Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 23:20:20,322 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 48812, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 23:20:20,322 [ERROR] Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-10 23:21:45,595 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 23:21:45,596 [INFO] InsultFilterService_0 registered at PYRO:obj_f5e9bb23afd1434192bd8b7e73f5b0d0@localhost:50152
2025-05-10 23:21:45,599 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 40762, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 23:21:45,601 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f284b65c250>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f284b65c250> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 23:21:45,608 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f284b65c250> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:21:45,608 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f284b65c250> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:21:45,608 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f284b65c250> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:21:45,609 [INFO] Created channel=1
2025-05-10 23:24:42,201 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-10 23:24:42,203 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 43160, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-10 23:24:42,205 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f05d85afe90>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f05d85afe90> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-10 23:24:42,209 [INFO] InsultFilterService_0 registered at PYRO:obj_ba1da2aa1fb24101ae4c72172ae0469d@localhost:50152
2025-05-10 23:24:42,211 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f05d85afe90> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:24:42,211 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f05d85afe90> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:24:42,211 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f05d85afe90> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-10 23:24:42,212 [INFO] Created channel=1
2025-05-11 10:18:05,909 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-11 10:18:05,911 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 35018, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:18:05,916 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f93c038ffd0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f93c038ffd0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-11 10:18:05,920 [INFO] InsultFilterService_0 registered at PYRO:obj_c1e27ccc42314857919b638562c7ddd9@localhost:50152
2025-05-11 10:18:05,928 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f93c038ffd0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:18:05,928 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f93c038ffd0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:18:05,929 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f93c038ffd0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:18:05,930 [INFO] Created channel=1
2025-05-11 10:20:55,035 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-11 10:20:55,035 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 58446, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:20:55,037 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f5876efc5d0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f5876efc5d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-11 10:20:55,039 [INFO] InsultFilterService_0 registered at PYRO:obj_69390759e75042798d39edde2bba6641@localhost:50152
2025-05-11 10:20:55,041 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f5876efc5d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:20:55,041 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f5876efc5d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:20:55,041 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f5876efc5d0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:20:55,042 [INFO] Created channel=1
2025-05-11 10:23:46,908 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-11 10:23:46,909 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 38118, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:23:46,910 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9cc5dfc350>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9cc5dfc350> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-11 10:23:46,911 [INFO] InsultFilterService_0 registered at PYRO:obj_2ba95791f8b64b3ca6441ebd476e6981@localhost:50152
2025-05-11 10:23:46,915 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9cc5dfc350> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:23:46,916 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9cc5dfc350> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:23:46,916 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9cc5dfc350> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:23:46,916 [INFO] Created channel=1
2025-05-11 10:27:55,559 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-11 10:27:55,560 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 34988, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:27:55,562 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f635c300310>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f635c300310> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-11 10:27:55,566 [INFO] InsultFilterService_0 registered at PYRO:obj_ac7d1d5bc9494348b14026515bdad64a@localhost:50152
2025-05-11 10:27:55,568 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f635c300310> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:27:55,568 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f635c300310> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:27:55,569 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f635c300310> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:27:55,574 [INFO] Created channel=1
2025-05-11 10:28:16,112 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-11 10:28:16,113 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 54864, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:28:16,114 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f6e6023c150>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f6e6023c150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-11 10:28:16,118 [INFO] InsultFilterService_0 registered at PYRO:obj_df6d354309104675ae1777fbcb53bdf1@localhost:50152
2025-05-11 10:28:16,119 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f6e6023c150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:28:16,119 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f6e6023c150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:28:16,120 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f6e6023c150> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:28:16,120 [INFO] Created channel=1
2025-05-11 10:29:04,129 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-11 10:29:04,130 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 38728, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:29:04,131 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7efddcdc4090>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7efddcdc4090> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-11 10:29:04,134 [INFO] InsultFilterService_0 registered at PYRO:obj_0e8c8f0b4ea64b609283e15aeb111f7e@localhost:50152
2025-05-11 10:29:04,137 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7efddcdc4090> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:29:04,137 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7efddcdc4090> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:29:04,137 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7efddcdc4090> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:29:04,138 [INFO] Created channel=1
2025-05-11 10:29:44,576 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-11 10:29:44,577 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 35214, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:29:44,578 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f3bf6688090>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f3bf6688090> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-11 10:29:44,582 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f3bf6688090> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:29:44,582 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f3bf6688090> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:29:44,582 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f3bf6688090> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:29:44,583 [INFO] Created channel=1
2025-05-11 10:29:44,584 [INFO] InsultFilterService_0 registered at PYRO:obj_f60809f725bd4246a0dece16f8e1ac32@localhost:50152
2025-05-11 10:30:34,994 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-11 10:30:34,995 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 44082, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:30:34,996 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f559321ff10>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f559321ff10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-11 10:30:35,000 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f559321ff10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:30:35,000 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f559321ff10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:30:35,000 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f559321ff10> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:30:35,000 [INFO] InsultFilterService_0 registered at PYRO:obj_3bfa3505cdfc4847a5d92899868d88a5@localhost:50152
2025-05-11 10:30:35,001 [INFO] Created channel=1
2025-05-11 10:31:22,502 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-11 10:31:22,503 [ERROR] Socket failed to connect: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 44088, 0, 0)>; error=111 (Connection refused)
2025-05-11 10:31:22,503 [ERROR] TCP Connection attempt failed: ConnectionRefusedError(111, 'Connection refused'); dest=(<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 5672, 0, 0))
2025-05-11 10:31:22,503 [ERROR] AMQPConnector - reporting failure: AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused')
2025-05-11 10:31:22,503 [INFO] Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
2025-05-11 10:31:22,504 [ERROR] Socket failed to connect: <socket.socket fd=10, family=2, type=1, proto=6, laddr=('127.0.0.1', 39110)>; error=111 (Connection refused)
2025-05-11 10:31:22,504 [ERROR] TCP Connection attempt failed: ConnectionRefusedError(111, 'Connection refused'); dest=(<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('127.0.0.1', 5672))
2025-05-11 10:31:22,504 [ERROR] AMQPConnector - reporting failure: AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused')
2025-05-11 10:31:22,504 [ERROR] AMQP connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused'); first exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused').
2025-05-11 10:31:22,504 [ERROR] AMQPConnectionWorkflow - reporting failure: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused'); first exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused')
2025-05-11 10:31:22,504 [ERROR] Connection workflow failed: AMQPConnectionWorkflowFailed: 2 exceptions in all; last exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused'); first exception - AMQPConnectorSocketConnectError: ConnectionRefusedError(111, 'Connection refused')
2025-05-11 10:31:22,504 [ERROR] Error in _create_connection().
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/pika/adapters/blocking_connection.py", line 451, in _create_connection
    raise self._reap_last_connection_workflow_error(error)
pika.exceptions.AMQPConnectionError
2025-05-11 10:32:04,688 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-11 10:32:04,692 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 59260, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:32:04,693 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f1d0718ff50>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f1d0718ff50> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-11 10:32:04,694 [INFO] InsultFilterService_0 registered at PYRO:obj_386ed43ce9f14284b6d60e8fc3b23e36@localhost:50152
2025-05-11 10:32:04,698 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f1d0718ff50> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:32:04,699 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f1d0718ff50> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:32:04,699 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f1d0718ff50> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:32:04,699 [INFO] Created channel=1
2025-05-11 10:39:27,378 [INFO] InsultFilterService_0 registered at PYRO:obj_dfaa6d0b7d864c74bcfdb37a63384120@localhost:50152
2025-05-11 10:39:27,390 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-11 10:39:27,391 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 33112, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:39:27,393 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7faae99a0550>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7faae99a0550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-11 10:39:27,399 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7faae99a0550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:39:27,400 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7faae99a0550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:39:27,401 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7faae99a0550> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:39:27,402 [INFO] Created channel=1
2025-05-11 10:41:23,092 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 33112, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:41:23,093 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 33112, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:41:23,094 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 33112, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:41:23,094 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-11 10:41:23,095 [INFO] Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-11 10:41:23,096 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 33112, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:41:23,096 [ERROR] Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-11 10:50:52,714 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-11 10:50:52,715 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 60044, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:50:52,717 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7feec2590250>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7feec2590250> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-11 10:50:52,718 [INFO] InsultFilterService_0 registered at PYRO:obj_e1f276b20b5c40379d467b2b5956e51f@localhost:50152
2025-05-11 10:50:52,722 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7feec2590250> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:50:52,722 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7feec2590250> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:50:52,722 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7feec2590250> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 10:50:52,723 [INFO] Created channel=1
2025-05-11 10:53:21,919 [INFO] Aborting transport connection: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 60044, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:53:21,920 [INFO] _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 60044, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:53:21,920 [INFO] Deactivating transport: state=1; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 60044, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:53:21,921 [INFO] AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-11 10:53:21,921 [INFO] Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-11 10:53:21,922 [INFO] Closing transport socket and unlinking: state=3; <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 60044, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 10:53:21,922 [ERROR] Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
2025-05-11 11:01:08,117 [INFO] InsultFilterService_0 registered at PYRO:obj_4ef8d34477024edab4281c00a1eeb58b@localhost:50152
2025-05-11 11:01:08,116 [INFO] Pika version 1.3.2 connecting to ('::1', 5672, 0, 0)
2025-05-11 11:01:08,119 [INFO] Socket connected: <socket.socket fd=10, family=10, type=1, proto=6, laddr=('::1', 43636, 0, 0), raddr=('::1', 5672, 0, 0)>
2025-05-11 11:01:08,120 [INFO] Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fc061dfc050>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fc061dfc050> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
2025-05-11 11:01:08,126 [INFO] AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fc061dfc050> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 11:01:08,126 [INFO] AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fc061dfc050> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 11:01:08,127 [INFO] Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fc061dfc050> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
2025-05-11 11:01:08,127 [INFO] Created channel=1
